$NetBSD: patch-pyproject.toml,v 1.1 2024/12/29 22:34:36 leot Exp $

Add missing version.

This is required by newer maturin.

Addressed upstream via a different patch.

--- pyproject.toml.orig	2024-12-29 22:22:12.797107590 +0000
+++ pyproject.toml
@@ -4,6 +4,7 @@ build-backend = "maturin"
 
 [project]
 name = "mitmproxy_rs"
+version = "0.5.2"
 requires-python = ">=3.10"
 classifiers = [
     "Programming Language :: Rust",
